src/main/java/Alex/Alex.java

```diff
@@ -0,0 +1,161 @@
+package Alex;
+
+import Alex.exception.DukeException;
+import Alex.manager.TaskManager;
+import Alex.task.Deadline;
+import Alex.task.Event;
+import Alex.task.Task;
+import Alex.task.Todo;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class Alex {
+    public static void main(String[] args) {
+        TaskManager taskManager = new TaskManager();
+        Scanner myObj = new Scanner(System.in);
+        greeting();
+        /** Constantly asks for user input until bye is received */
+        while(!taskManager.getExit()) {
+            String userInput = myObj.nextLine();
+            try {
+                handleInput(userInput, taskManager);
+            } catch (IndexOutOfBoundsException e) {
+                System.out.println("☹ OOPS!!! The description of a " + e.getMessage() + " cannot be empty.");
+                printLine();
+            } catch (DukeException e) {
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                printLine();
+            }
+
+            }
+
+    }
+    public static void echoResponse(Task task, int number) {
+        System.out.println("Got it. I've added this task:" + "\n " + task + "\n" +
+                    "Now you have " + number + " tasks in the list" );
+        printLine();
+    }
+    public static void printLine() {
+        String horizontalLine = "____________________________________________________________";
+        System.out.println(horizontalLine);
+    }
+    public static void greeting() {
+        printLine();
+        String greet = "Hello! I'm Alex\nWhat can I do for you today?";
+        System.out.println(greet);
+        printLine();
+    }
+    public static void exit() {
+        String exit = "Bye. Hope to chat with you again soon!";
+        System.out.println(exit);
+        printLine();
+    }
+    /** Method that utilises a taskManager to handle different user inputs accordingly */
+    public static void handleInput(String userInput, TaskManager taskManager) throws DukeException {
+        String[] words = userInput.split(" ");
+        String[] allCommands = new String[]{"todo" , "bye", "event", "list" , "unmark" , "mark", "deadline"};
+        List<String> commandList = new ArrayList<>(Arrays.asList(allCommands));
+        String command = words[0].toLowerCase();
```

Comment:
Try to avoid magic literals. There has been a few more instances in the rest of your code. You can use a named constant makes the code easier to understand because the name tells us more about the meaning of the number.